src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/McpServer/McpServer-CSharp/McpServer-CSharp.csproj.in

```diff
@@ -1,15 +1,35 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
+    <TargetFramework Condition="'$(TargetFrameworkOverride)' == ''">net9.0</TargetFramework>
+    <TargetFramework Condition="'$(TargetFrameworkOverride)' != ''">TargetFrameworkOverride</TargetFramework>
+    <!--#if (SelfContained || NativeAot)-->
+    <RuntimeIdentifiers>win-x64;win-arm64;osx-arm64;linux-x64;linux-arm64</RuntimeIdentifiers>
```

Comment:
I think it might help to have a code comment here about RID selection (briefly alluding to the trade-off or linking to a good doc)

Reply:
Yes, essentially, there's a decision here that the package author needs to make (either by default or with some extra knowledge they have): _what RIDs to target_. The list can be modified by the package author but it will impact who can consume the MCP server. A small comment to call out that this is a decision point would help, I think.